propagators/opentelemetry-propagator-aws-xray/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "opentelemetry-propagator-aws-xray",
```

Comment:
```suggestion
  "name": "@opentelemetry/propagator-aws-xray",
```

Reply:
@open-telemetry/javascript-maintainers do we want to have another namespace or even just a naming convention for vendor-specific and contrib things? Maybe `@otel-contrib/propagator-aws-xray` or `@opentelemetry/contrib-propagator-aws-xray`?

Reply:
No problem, which one should it be changed? ```@opentelemetry/contrib-propagator-aws-xray``` or ```@otel-contrib/propagator-aws-xray```

Reply:
For now, lets do `@opentelemetry/propagator-xray`. I will bring up the naming discussion at tomorrow's SIG meeting.

Reply:
Got it! Thank you so much!